docs/eventing/channels/channels-crds.md

```diff
@@ -13,13 +19,25 @@ procedure:
 
 This is a non-exhaustive list of the available Channels for Knative Eventing.
 
-!!! note
-    Inclusion in this list is not an endorsement, nor does it imply any level of support.
+Notes:
+
+* Inclusion in this list is not an endorsement, nor does it imply any level of
+  support.
```

Comment:
same re notes, the `!!! note` format is correct, see https://github.com/knative/docs/blob/main/contribute-to-docs/getting-started/formatting.md#admonitions